src/py42/services/cases.py

```diff
@@ -118,4 +127,6 @@ def update(
         except Py42BadRequestError as err:
             if u"NO_EDITS_ONCE_CLOSED" in err.response.text:
                 raise Py42UpdateClosedCaseError(err)
+            if u"DESCRIPTION_TOO_LONG" in err.response.text:
```

Comment:
Same goes here - `elif` instead of `if`.  

Here is why: Say this code changes and no longer raised the error now, we may forget to change it to elif then. Having it now is more readable as well.

Reply:
Added.